microsoft-edge/webview2/release-notes/index.md

```diff
@@ -85,6 +85,232 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
+   * [ICoreWebView2ExperimentalEnvironmentOptions2::put_ScrollBarStyle](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironmentoptions2?view=webview2-1.0.2470-prerelease&preserve-view=true#put_scrollbarstyle)
+
+* [COREWEBVIEW2_SCROLLBAR_STYLE enum](/microsoft-edge/webview2/reference/win32/webview2experimental-idl?view=webview2-1.0.2470-prerelease#corewebview2_scrollbar_style)
+
+---
+
+
+<!-- ---------- -->
+* Updated the WebMessageObjects API to allow injecting DOM objects into WebView2 content that's constructed via the app and via the `CoreWebView2.PostWebMessage` API in the other direction.  Added a new web object type to represent a file system handle that can be posted to the web content to provide it with filesystem access.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2` Class:
+   * [CoreWebView2.PostWebMessageAsJsonWithAdditionalObjects Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.postwebmessageasjsonwithadditionalobjects?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)
+
+* `CoreWebView2Environment` Class:
+   * [CoreWebView2Environment.CreateWebFileSystemDirectoryHandle Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createwebfilesystemdirectoryhandle?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)
+   * [CoreWebView2Environment.CreateWebFileSystemFileHandle Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createwebfilesystemfilehandle?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)
+
+* `CoreWebView2FileSystemHandle` Class:
+   * [CoreWebView2FileSystemHandle.Kind Property](/dotnet/api/microsoft.web.webview2.core.corewebview2filesystemhandle.kind?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)
+   * [CoreWebView2FileSystemHandle.Path Property](/dotnet/api/microsoft.web.webview2.core.corewebview2filesystemhandle.path?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)
+   * [CoreWebView2FileSystemHandle.Permission Property](/dotnet/api/microsoft.web.webview2.core.corewebview2filesystemhandle.permission?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)
+
+* [CoreWebView2FileSystemHandleKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2filesystemhandlekind?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)
+   * `File`
+   * `Directory`
+
+* [CoreWebView2FileSystemHandlePermission Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2filesystemhandlepermission?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)
+   * `ReadOnly`
+   * `ReadWrite`
+
+* `CoreWebView2ObjectCollection` Class:
+   * [CoreWebView2ObjectCollection.InsertValueAtIndex Property](/dotnet/api/microsoft.web.webview2.core.corewebview2objectcollection.insertvalueatindex?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)
```

Comment:
not found for .NET:
* `CoreWebView2ObjectCollection` Class:
   * [CoreWebView2ObjectCollection.InsertValueAtIndex Property](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2objectcollection.insertvalueatindex?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)
   * [CoreWebView2ObjectCollection.RemoveValueAtIndex Property](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2objectcollection.removevalueatindex?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true)

eg near: 
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2filesystemhandlepermission?view=webview2-dotnet-1.0.2470-prerelease&preserve-view=true

They are found for other platforms: 

WinRT:
* `CoreWebView2ObjectCollection` Class:
   * [CoreWebView2ObjectCollection.InsertValueAtIndex Property](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2objectcollection?view=webview2-winrt-1.0.2470-prerelease&preserve-view=true#insertvalueatindex)
   * [CoreWebView2ObjectCollection.RemoveValueAtIndex Property](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2objectcollection?view=webview2-winrt-1.0.2470-prerelease&preserve-view=true#removevalueatindex)

Win32:
* [ICoreWebView2ExperimentalObjectCollection](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2experimentalobjectcollection?view=webview2-1.0.2470-prerelease&preserve-view=true)
   * [ICoreWebView2ExperimentalObjectCollection::InsertValueAtIndex](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2experimentalobjectcollection?view=webview2-1.0.2470-prerelease&preserve-view=true#insertvalueatindex)
   * [ICoreWebView2ExperimentalObjectCollection::RemoveValueAtIndex](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2experimentalobjectcollection?view=webview2-1.0.2470-prerelease&preserve-view=true#removevalueatindex)

@peiche-jessica @victorhuangwq

Reply:
kept for win32 only